src/views/Visualizer/View/View.js

```diff
@@ -214,6 +214,8 @@ export class Visualizer extends React.Component {
     const { layersTreeState, view } = this.props;
     const { legends } = this.state;
 
+    const { map: { customStyle: { layers: customStyleLayers } } } = view;
```

Comment:
What happens if `view` of `view.map` or `view.map.customStyle` are `undefined`?
Shouldn't we have default values (` = {}`) in destructuring to avoid error cases?